setup.py

```diff
@@ -23,11 +23,11 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-image-explorer',
-    version='2.0',
+    version='2.1',
```

Comment:
I think it good time to add path version as well
`2.1.0`

Reply:
done